tomcat8.0/build.gradle

```diff
@@ -0,0 +1,63 @@
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.ClassVisitor
+import org.objectweb.asm.ClassWriter
+import org.objectweb.asm.MethodVisitor
+import org.objectweb.asm.Opcodes
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'org.ow2.asm:asm:5.1'
+    }
+}
+
+dependencies {
+    // Tomcat
+    [ 'tomcat-embed-core', 'tomcat-embed-jasper', 'tomcat-embed-el' ].each {
+        compile "org.apache.tomcat.embed:$it:8.5.6"
+    }
+}
+
+managedDependencies {
+    // Replace commons-logging (Tomcat's logging API) with SLF4J.
+    runtime 'org.slf4j:jcl-over-slf4j'
+}
+
+// Remove the methods that did not exist until Tomcat 8.5 from the ProtocolHandler implementation using
+// bytecode manipulation. We need the methods to exist during compilation time since we compile the
```

Comment:
IIRC we do not need this hack anymore because we can build against Tomcat 8.0 for `armeria-tomcat8.0`.